webapp/src/containers/Rhs/index.tsx

```diff
@@ -13,28 +13,23 @@ import ProjectList from './projectList';
 import ProjectDetails from './projectDetails';
 
 const Rhs = (): JSX.Element => {
-    const {state, makeApiRequest, getApiState} = usePluginApi();
+    const {state, getApiState} = usePluginApi();
     const {isConnected} = getWebsocketEventState(state);
     const {isLoading} = getApiState(pluginConstants.pluginApiServiceConfigs.getUserDetails.apiServiceName);
 
-    // Check if user is connected on page reload
-    useEffect(() => {
-        makeApiRequest(pluginConstants.pluginApiServiceConfigs.getUserDetails.apiServiceName);
-    }, []);
+    if (isLoading) {
+        return <LinearLoader/>;
+    }
 
     return (
         <div
             id='scrollableArea'
-            className='overflow-auto height-rhs position-relative padding-16'
         >
-            {isLoading && <LinearLoader/>}
-            {!isLoading && !isConnected && <AccountNotLinked/>}
-            {
-                !isLoading && isConnected && (
-                    getProjectDetailsState(state).projectID ?
-                        <ProjectDetails {...getProjectDetailsState(state)}/> :
-                        <ProjectList/>)
-            }
+            {!isConnected && <AccountNotLinked/>}
+            {isConnected && (
+                getProjectDetailsState(state).projectID ?
+                    <ProjectDetails {...getProjectDetailsState(state)}/> :
+                    <ProjectList/>)}
```

Comment:
can we use the ternary operator here?

Reply:
no as nested ternary not allowed by lint here